docs/nada-run.md

```diff
@@ -1,9 +1,9 @@
-# program-simulator
+# nada-run
 
-The program-simulator tool simulates program execution against a local stripped down version of a Nillion cluster
+The nada-run tool locally executes a program against a local stripped down version of a Nillion cluster
```

Comment:
```suggestion
The nada-run tool executes a nada program using the Nada VM with the cryptographic algorithms but without the p2p network.
```

Reply:
We have to move from cluster terminology to subnets and I think this is a better description.